go/vt/orchestrator/inst/durability.go

```diff
@@ -161,3 +177,46 @@ func (d *durabilityCrossCell) replicaSemiSync(master, replica *topodatapb.Tablet
 	}
 	return false
 }
+
+//=======================================================================
+
+type durabilitySpecified struct {
+	promotionRules map[string]CandidatePromotionRule
+}
+
```

Comment:
Can you add some comments describing what this does?

Reply:
Added comments in the last commit.